src/pythonic/index.md

```diff
@@ -174,6 +174,9 @@ class Sieve:
         primal.sieve_destroy(self.sieve)
 
     def is_prime(self, n):
+        if n > self.upper_bound():
+            raise IndexError("{} not in upper bound {}"
```

Comment:
I know most of the standard library uses a `# Panics` section when a particular call may panic. So it sounds like a good idea to add that to the docs for the Rust `sieve_is_prime`.

I like that wording. What about saying "A Python `Exception` is preferable over rust panicking and unwinding, so `is_prime` checks if `n` exceeds `upper_bound`."? That way you make it more explicit that we're translating an error from Rust to use the idiomatic Python mechanism.